panther_lights/config/led_config.yaml

```diff
@@ -0,0 +1,123 @@
+panels:
+  - channel: 1
+    number_of_leds: 46
+  - channel: 2
+    number_of_leds: 46
+
+segments:
+  - name: front
+    channel: 1
+    led_range: 0-45
+  - name: rear
+    channel: 2
+    led_range: 45-0
```

Comment:
I see this because the panels are identical in hardware and one is reversed relative to the other - so the first one left LED on the rear panel is 45. Am I correct?

Reply:
Yes, one of the segments is reversed to make panels not reverset relative to each other